drivers/ina3221/alerts.c

```diff
@@ -34,13 +34,24 @@ int _ina3221_enable_alert(ina3221_t *dev, ina3221_alert_t alert,
     }
     dev->alert_callbacks[alert] = cb;
     dev->alert_callback_arguments[alert] = arg;
-    int check = gpio_init_int(
-        dev->params.alert_pins[alert],
-        GPIO_IN,
-        GPIO_FALLING,
-        cb,
-        arg
-        );
+    int check;
+    if (dev->params.gpio_config & (1 << alert)) {
+        check = gpio_init_int(
+            dev->params.alert_pins[alert],
+            GPIO_IN_PU,
```

Comment:
Maybe just using `(dev->params.gpio_config & (1 << alert)) ? GPIO_IN_PU : GPIO_IN` for the second parameter of `gpio_init_int()` is easier to read, as this would get rid of the `if`...`else` case handling?

Reply:
Yes, that´s better.